pkg/network/netlink/message.go

```diff
@@ -0,0 +1,41 @@
+package netlink
+
+import (
+	"syscall"
+
+	"github.com/mdlayher/netlink"
+	"github.com/mdlayher/netlink/nlenc"
+)
+
+// Copyright (C) 2016-2021 Matt Layher
+// Source https://github.com/mdlayher/netlink/message.go
```

Comment:
This link as-is doesn't work. I suggest linking to a specific commit and line, like so: `https://github.com/mdlayher/netlink/blob/00ca4099ac0015352f430fd975b7977852a009f6/message.go#L249`

Otherwise, using `https://github.com/mdlayher/netlink/blob/master/message.go` would work.

Reply:
Good catch. I'll update it